python/fledge/services/core/api/health.py

```diff
@@ -208,7 +207,7 @@ async def get_storage_health(request: web.Request) -> web.Response:
         response['disk']['available'] = available
         response['disk']['status'] = status
     except Exception as ex:
-        msg = "Failed to get disk stats! {}".format(str(ex))
+        msg = "Failed to get disk stats for storage service due to {}".format(str(ex))
```

Comment:
due to can be removed. Simply `Failed to <action e.g. get disk stats>  for <if specific>!` Followed by str(ex)